index.rst

```diff
@@ -6,20 +6,423 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
+Overview
+========
+
+This document describes the procedure to pump and cool down LATISS from ambient pressure to its operational temperatures. 
+It also outlines how to perform a controlled warm-up in the case of servicing or programmed power cuts. 
+
+Pumping LATISS to vacuum
+========================
+
+Under atypical circumstances, LATISS needs to be warmed up to ambient temperature. 
+Large periods of power outage, severe earthquakes (`and different levels of recovery based on severity <https://tstn-027.lsst.io>`__), or technical intervention may have necessitated warm-up.
+Pumping down, cooling, and powering up LATISS takes at least two days, and must be organized by ticket on the summit calendar and supported by daytime engineering support. 
+Mario Rivera, Craig Lage, and Alysha Shugart are points of contact for understanding and initiating the procedure. 
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
+The safety team is responsible for the the approval of the LOTO procedure, which will be linked on this page once it is affirmed.
+ 
+Start by powering up the instrument cooling electronics, located in the first floor cabinet. 
```

Comment:
I'm worried that someone might include the cryotiger (poly cold) as "instrument cooling electronics" .. maybe rephrase to "powering up the instruments required to support the instrument cooldown"

Reply:
In fact, you might want to add some sort of `..important::` note that says the instrument can be damaged by prematurely powering on the polycold system and extreme caution must be exercised to ensure everything is ready before starting to cool.

Reply:
👍